victor_fake_hardware_interface/src/victor_fake_hardware_interface/minimal_fake_arm_interface.py

```diff
@@ -84,58 +84,43 @@ def join_feedback_threads(self):
         self.gripper_status_thread.join()
 
 
-class MinimalFakeArmInterface:
-    cartesian_field_names = ["x", "y", "z", "a", "b", "c"]
-    joint_names = ['joint_' + str(i) for i in range(1, 8)]
+class MinimalFakeControlModeInterface:
```

Comment:
Github's diff is not great here, but I gather you:
1) Defined a `MinimalFakeControlModeInterface`
2) Refactored `MinimalFakeArmInterface` to use the `MinimalFakeControlModeInterface`

That parts seems fine, let me know if something else changed.